src/mito/src/engine.rs

```diff
@@ -57,8 +57,8 @@ fn region_id(table_id: TableId, n: u32) -> RegionId {
 }
 
 #[inline]
-fn table_dir(schema_name: &str, table_name: &str) -> String {
-    format!("{}/{}/", schema_name, table_name)
+fn table_dir(schema_name: &str, table_name: &str, table_id: &str) -> String {
```

Comment:
Using `TableId` type for `table_id` directly. No need to  call`table_id.to_string()`.

Reply:
Thanks